.clang-tidy

```diff
@@ -1,6 +1,7 @@
 ---
 Checks: '-*,
   abseil-*,
+  -abseil-no-namespace,  # https://bugs.llvm.org/show_bug.cgi?id=47947
```

Comment:
I think this comment could be a problem because that comment is in the string and it will affect the following checks. If so, how about moving this comment out of string? For example,

```
# Disable abseil-no-namespace: https://bugs.llvm.org/show_bug.cgi?id=47947
Check:
```

Reply:
Done.